src/main/webapp/index-script.js

```diff
@@ -11,31 +11,56 @@ function loadPage() {
 
 /* Activates functionality for search bar and log-in button. */
 function loadMainButtons() {
-  const head = document.getElementsByTagName('head');
-  const clearIcon = document.querySelector(".clear-icon");
-  const searchIcon = document.querySelector("#search-icon");
-  const searchForm = document.querySelector("#searchForm");
-  const searchBar = document.querySelector(".search");
+  activateSearchBar();
+  activateTutorial();
+
   const logInButton = document.querySelector("#login");
-  const closeTutorial = document.querySelector("#exit");
-  const prev = document.querySelector(".prev");
-  const next = document.querySelector(".next");
-  const tutorialText = document.getElementById("centralText");
+  // Calls login servlet onclick.
+  logInButton.addEventListener("click", () => {
+    window.location.href = "/login";
+  }); 
+
   const modalClosers = document.querySelectorAll('[data-modal-close-button]');
+  // Button to close the modal and deactiviate overlay
+  modalClosers.forEach(button => {
```

Comment:
It seems like every PR you have been deleting/replacing these same lines of code.